torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2272,6 +2272,29 @@ def generator():
 
     return list(generator())
 
+def sample_inputs_max_pool2d(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    kerneli = [[3, 2], [3, 3]]
+    stridei = [[2, 2]]
+    Ni = [8, 16]
+    Ci = [32]
+    Hi = [8, 16]
+    Wi = [8, 16]
+    ceil_modei = [True, False]
+    paddingi = [0, 1]
+    dilationi = [1, (1, 2)]
+
+    def generator():
+        for kernel, stride, N, C, H, W, ceil_mode, padding, dilation in product(kerneli, stridei, Ni, Ci, Hi, Wi, ceil_modei, paddingi, dilationi):
+            max_pool = torch.nn.MaxPool2d(kernel, stride, ceil_mode=ceil_mode, padding=padding, dilation=dilation)
+
+            yield SampleInput(make_arg((N, C, H, W)), args=(max_pool.kernel_size, max_pool.stride,
```

Comment:
Although the ordering for positional args is correct here, I don't think it's a good idea to pass all of the params as positional arguments as mentioned #62545.

Reply:
do you mean as keyword arguments ?

Reply:
Yeah I mean it would be better to add keyword arguments since positional args ordering is not correct.

Reply:
Good suggestion! Done